infra/load-balancer.tf

```diff
@@ -0,0 +1,25 @@
+resource "helm_release" "load_balancer_controller" {
+  name       = "jovand-loadbalancer-controller"
+  repository = "https://aws.github.io/eks-charts"
+  chart      = "aws-load-balancer-controller"
+  namespace  = "default"
+  set {
+    name  = "clusterName"
+    value = module.eks.cluster_name
+  }
+
+  set {
+    name  = "serviceAccount.create"
+    value = "false"
```

Comment:
This should be removed as by default it's true and that is exactly what we need in order to map k8s svc account to AWS IAM role